app.rb

```diff
@@ -18,6 +18,7 @@
   c.delete :legislatures
 
   c[:name] = c[:country]
+  c[:code] = c[:code]
```

Comment:
Was this really necessary? Or is this a leftover artifact of you playing with something else here at some point?